### PR TITLE
feat(erc-20): CWC-355 Add ERC-20 (VLX) Token

### PR DESCRIPTION
**Pull request by srt1678**
Add ERC-20 (VLX) Token

### CODE REVIEW

**Review by hank121314 — APPROVED** [review]

**Review by hankliu418 — DISMISSED** [review]

File: packages/coin-eth/src/config/tokenType.ts [hunk 1]
```diff
@@ -1,4 +1,12 @@
 export const TOKENTYPE = [
+  //VLX
+  {
+    name: "VLX",
```
Comment (delbinwen) [inline packages/coin-eth/src/config/tokenType.ts:4]: not sure we shall name it Velas or VLX. to confirm with PO

Comment (delbinwen) [reply packages/coin-eth/src/config/tokenType.ts:4]: confirmed. please update the name to `Velas`

**Comment by delbinwen** [comment]
suggested scope is `erc-20`, so that the title shall be feat(erc-20): ...

**Review by hank121314 — APPROVED** [review]

**Review by delbinwen — APPROVED** [review]
ok

**Pull request merged**